src/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,83 @@
+// REACT
+import * as React from 'react';
+// STORYBOOK
+import { storiesOf } from '@storybook/react';
+import { select, text } from '@storybook/addon-knobs';
+// VENDOR
+import styled, { ThemeProvider } from '@xstyled/styled-components';
+import { th } from '@xstyled/system';
+// ANCHOR
+import { RootTheme } from '../../src/theme';
+import { Tabs } from './Tabs.component';
+import { Position, Variant } from './utils';
+
+const { Pane } = Tabs;
+
+const StyledStory = styled('div')`
+    padding: 2rem 1rem;
+    color: text.body;
+    font-family: ${th('typography.fontFamily')};
+`;
+
+storiesOf('Components/Tabs', module)
+    .add('Default', () => (
+        <ThemeProvider theme={RootTheme}>
+            <StyledStory>
+                <Tabs
+                    variant={select<Variant>(
+                        'variant',
+                        ['regular', 'tabular'],
+                        'regular'
+                    )}
+                    position={select<Position>(
+                        'position',
+                        ['top', 'bottom', 'right', 'left'],
+                        'top'
+                    )}
+                    activeKey={text('activeKey', '') || undefined}
+                    defaultActiveKey={text('defaultActiveKey', '') || undefined}
+                >
+                    <Pane title="Banana" key="banana">
+                        Best when just starting to brown.
+                    </Pane>
+                    <Pane title="Orange" key="orange">
+                        Juicy and delicious, generally underrated.
+                    </Pane>
+                    <Pane title="Apple" key="apple">
+                        Only okay.
+                    </Pane>
+                    <Pane title="Strawberry" key="strawberry" disabled>
+                        Trivia: Strawberries are the only known red fruit.
```

Comment:
Wait until you find out about cherries, cranberries, guava, red grapes, watermelon, pomegranate, strawberries, raspberries, red apple (with skin). 😏

Reply:
watermelons are green

Reply:
tomatoes are red